libs/components/core/src/lib/modules/help/help-update-args.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ *  Options for updating global help.
+ */
+export interface SkyHelpUpdateArgs {
+  /**
+   * A unique key that identifies the current help content to display. If not provided, the page's default help content will be displayed.
```

Comment:
The consumer would still need to set the `helpKey` to `undefined`, yes? Should the second sentence start with "If set to `undefined`, the..."?

Reply:
Good call. I wrote that sentence before tweaking that. Done.